contracts/SBCDepositContract.sol

```diff
@@ -299,6 +299,17 @@ contract SBCDepositContract is
         bool _unwrapToGNO,
         uint256 gasLimit
     ) internal returns (bool success) {
+        // Skip withdrawal that burns tokens to avoid a revert condition
+        // https://github.com/OpenZeppelin/openzeppelin-contracts/blob/dad73159df3d3053c72b5e430fa8164330f18068/contracts/token/ERC20/ERC20.sol#L278
+        if (_receiver == address(0)) {
+            return true;
+        }
+
+        // Skip withdrawal for 0 tokens
+        if (_amount == 0) {
```

Comment:
Does it cause the process to revert? If not, possibly we should process it.

Reply:
With current tokens there's no condition that can cause a revert. However what's the point of processing a withdrawal for 0 value? A system transaction does not have any other result than a change in state which won't happen

Reply:
I agree that no state changes will happen.
My suggestion was only that we can remove this extra conditions if they do not change anything (except of burned gas).